Managers/AudioMan.cpp

```diff
@@ -1,50 +1,60 @@
 #include "AudioMan.h"
 #include "ConsoleMan.h"
 #include "SettingsMan.h"
-#include "Sound.h"
-#include "RTEError.h"
+#include "SoundContainer.h"
 
 namespace RTE {
 	const std::string AudioMan::m_ClassName = "AudioMan";
 
 	// I know this is a crime, but if I include it in FrameMan.h the whole thing will collapse due to int redefinitions in Allegro
 	std::mutex g_SoundEventsListMutex[c_MaxClients];
```

Comment:
I've put it at the bottom of AudioMan.h, inside the namespace but not inside the class, since I don't think it was a class variable. Seem fine (assuming it works)?

Reply:
I've put this into the class and had no issues.
I guess only time will tell?

Reply:
Oh cool. Putting it in the header made it fail to build but it seems fine in the class.